base_search_custom_field_filter/models/base.py

```diff
@@ -61,6 +61,9 @@ def load_views(self, views, options=None):
             field = custom_filter._get_related_field()
             field_name = custom_filter.expression
             res["fields"][field_name] = field.get_description(self.env)
+            # Force name for avoiding error accesing to this field from pivot and graph
+            # (needed for owl framework)
+            res["fields"][field_name]["name"] = field_name
```

Comment:
Is it the technical one?

Reply:
yes, it corresponds to the expression used in the definition of the custom filter field